hive/src/main/java/io/confluent/connect/storage/hive/schema/TimeBasedSchemaGenerator.java

```diff
@@ -42,13 +42,17 @@ public TimeBasedSchemaGenerator() {
   }
 
   public TimeBasedSchemaGenerator(Map<String, Object> config) {
-    this.config = config;
+    this.config = config == null ? new HashMap<String, Object>() : config;
```

Comment:
This is given in the constructor. Do you expect the contents of the Map to ever change beyond this point?
My guts say no, in which case `Collections.emptyMap()` would be preferable. But if there's even one case where it would make sense to add values to an initially empty `config` (for testing or not), I'd definitely keep it as is.